presets/4.3/bnf/moblite_6.txt

```diff
@@ -0,0 +1,106 @@
+#$ TITLE: HappyModel Moblite 6 settings
+#$ FIRMWARE_VERSION: 4.2
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: BNF
+#$ STATUS: COMMUNITY
+#$ KEYWORDS: HappyModel, Moblite, bnf
+#$ AUTHOR: antony-weber
+#$ DESCRIPTION: Basic settings for a HappyModel Moblite 6 quadcopter
+#$ INCLUDE_DISCLAIMER: misc/disclaimer/en/vtxtable.txt
+#$ DISCUSSION: https://github.com/betaflight/firmware-presets/pull/136
+
+defaults
```

Comment:
i think `defaults nasave`. Otherwise, it can reboot, can't it?

Reply:
Yeah you're right. Pretty sure it reboots right after `defaults` so it needs to be `defaults nosave`